manual/2-populate-the-structure/templates/procedure/procedure-template-flat.json

```diff
@@ -68,16 +71,14 @@
           "examples": []
         }
       ],
-      "examples": [
-        "| Usage | Example | |:--------|:----------------------------------------| | **Use** | Create a codespace from a template | | Avoid | Creating a codespace from a template | | **Use** | Use source control in your codespace | | Avoid | Use source control | | **Use** | Collaborate with others in a codespace | | Avoid | Set up a live share session |"
-      ]
+      "examples": []
     },
     {
       "id": "PROC-SUBTITLE",
       "group": "STRUCT",
```

Comment:
As discussed in the meeting, the "group" field should be removed for all group IDs